extensions/amp-story/1.0/amp-story-desktop-one-panel.css

```diff
@@ -24,16 +24,14 @@
 
 .i-amphtml-story-desktop-one-panel amp-story-page {
   margin: auto !important;
-  aspect-ratio: var(--i-amphtml-story-page-large-desktop-ratio) !important;
-  height: calc(100vh - 148px) !important;
-  max-height: 752px !important;
-  width: auto !important; 
+  height: min(752px, calc(100vh - 148px)) !important;
+  width: min(447px, calc((100vh - 148px) * var(--i-amphtml-story-page-large-desktop-ratio))) !important; 
```

Comment:
can you add a comment to explain the 752/447 magic numbers? If it's bound to some other logic, like when this UI actually triggers, it'd be useful to have cross referencing comments